text/0010-gatsby-docs-localization

```diff
@@ -0,0 +1,104 @@
+
+* Open Source Contributors tend to be familiar with Git
+* No need to integrate with an external API and software (aside from GitHub)
+* People want to be listed as contributors of the React/Gatsby organization on GitHub, and it's an incentive to contribute.
+* Can use GitHub features such as Code Owners and pull requests as quality assurance
+
+### Drawbacks
+
+The main drawback of this approach is that the Gatsby docs currently live in a monorepo with the rest of the Gatsby source, which means we can't just do a copy of the entire website and use `git merge` like we do with React (and Vue).
+
+This is also an issue if we want translations to live under a path (e.g. `gatsbyjs.org/ja` rather than `ja.gatsbyjs.org`).
+
+Other drawbacks we learned from working on the React translation:
+
+* Need to maintain a custom bot and system to keep the translations up to date.
+* It's not a system that can be easily put into a plugin to allow others to use.
+
+### Mitigation
+
+I think this is a viable approach if we can:
+
+1. move the docs to their own repo and
+2. are okay with doing subdomains (`ja.gatsbyjs.org`) instead of paths (`gatsbyjs.org/ja`)
+
+Otherwise we might want to consider the second alternatives:
+
+## Alternative 2: Translations Platform
+
+[Crowdin]() is a SaaS that allows projects to source translations using a built-in web based editor. It's popular in the JS community being used for projects such as [Electron](https://crowdin.com/project/electron) and [Yarn](https://crowdin.com/project/yarn).
+
+### Advantages
+
+* Lots of features, such as [machine translation](https://support.crowdin.com/pre-translation-via-machine/) and [in-context localization](https://support.crowdin.com/in-context-localization/).
+* Has a builtin [GitHub integration](https://support.crowdin.com/github-integration/).
+* We can make an official plugin for Crowdin integration that other projects can use.
+
+### Drawbacks
+
+We had considered using Crowdin for the localization of the React docs but realized that Crowdin had some [major drawbacks](https://reactjs.org/blog/2019/02/23/is-react-translated-yet.html):
+
+* It has a steep learning curve and hard for new translators to get accustomed to.
+* The GitHub integration isn't customizable: it takes three hours to compile all the languages and just publishes something in a single directory.
+* Translation quality issues:
+  * Doesn't handle web markup well; sites translated with Crowdin sometimes have invalid markup
+  * (as far as I can tell) no way to verify that a section/page has to pass a quality check before it gets published
+  
+Other drawbacks of Crowdin:
+
+* No way to prioritize which pages should get translated first
+* Hard to attribution of translators
+* Paid program ($125/mo for the 'Bronze' Organization Plan)
+* Quality of English documentation is mediocre, which is not a good sign for a translation app (see links above).
```

Comment:
Yeah, sad story, but they will improve docs quality with the time

(it's not an argument) I can speak with developers and support team as a native speaker (Russian, Ukarinian)